walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc

```diff
@@ -160,6 +160,7 @@ this may not be advisable._
 . Select *Overview* on the left and find the *rhmi-lab-nodejs-order-frontend* in the list.
 . Wait for it to finish deploying.
 . Click the route listed beside the *rhmi-lab-nodejs-order-frontend* to open the *Order Entry System*. A login screen with the title *{user-username} Realm* is displayed.
+// TODO: Verify this... In my case, the user `evals03` was already logged in (the name in top-right corner), I assume because there was a session in RH-SSO. Or maybe instruct here to open in the private browser session (like in the verification step).
```

Comment:
That's interesting, but might be linked to the SSO session as you said. I think we recommend the private session by default to "avoid potential session issues"?

Reply:
Sure.
Change the `Click the route... ` item to:

```
Open the URL listed beside the *rhmi-lab-nodejs-order-frontend* in  a private browser session to open the *Order Entry System*. A login screen with the title *{user-username} Realm* is displayed.
``` 
?